src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: 50%;
+  display: inline-block;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :not(:last-child),
+.stars--5 > .stars__star {
```

Comment:
try to be consistent and use `:nth-child` in all cases
in that case it would be easier to read